transmart-server/grails-app/services/org/transmart/server/subscription/QuerySetSubscriptionMailService.groovy

```diff
@@ -0,0 +1,134 @@
+package org.transmart.server.subscription
```

Comment:
These files are now the only functional parts of `transmart-server`, while this module was meant for only bundling components. Please, move the subscription functionality to either a separate component (could be a grails plugin or a plain groovy library, but then you need to add some Spring annotations), or to, e.g., core-db. `transmart-notifications`, `transmart-email` or something.